packages/experimental/Switch/src/useSwitch.ts

```diff
@@ -31,7 +31,15 @@ export const useSwitch = (props: SwitchProps): SwitchInfo => {
   const focusRef = isDisabled ? null : componentRef;
 
   if (__DEV__ && defaultChecked !== undefined && checked !== undefined) {
-    console.warn('The props defaultChecked and checked are mutually exclusive. Use only one of the props, do not use both.');
+    console.warn("The props 'defaultChecked' and 'checked' are mutually exclusive. Use only one of the props, do not use both.");
+  }
+
+  if (__DEV__ && labelPosition === 'after' && (!!props.onText || !!props.offText)) {
```

Comment:
Alternatively, you can simply just check for `if (__DEV__ && labelPosition === 'after')`. The only condition we need to know to set these values to null is labelPosition

Reply:
If we had that, it would trigger a warning even if the user didn't set a onText/offText prop. So should we still warn them even if they're using the prop correctly?

Reply:
Setting on/off text to null should happen regardless of whether we're in release/dev. Otherwise we'll have odd behavior in release even though the odd behavior doesn't show up in dev. But the warning should only fire in dev.

Reply:
Oops, thanks for catching that!